components/ui/InfiniteScroll.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
+    <div
+      v-for="(item, index) in items.slice(0, lastShown)"
+      :id="index + '-' + keyGenerator(item)"
+      :key="keyGenerator(item)"
+      class="scrollable"
+    >
+      <slot v-bind="item" />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import ScrollSectionsMixin from '~/mixins/scrollBetweenSections'
+
+@Component
+export default class extends Mixins(ScrollSectionsMixin) {
+  @Prop(Array) items!: Array<any>
+  @Prop(Function) keyGenerator!: Function
+  @Prop({ type: Number, default: 5 }) minItems!: number
+
+  lastShown: number = Math.min(this.minItems, this.items.length)
+
+  activeSectionChanged () : void {
+    const current: number = this.activeSection ? parseInt(this.activeSection.split('-', 2)[0]) : 0
+    const oldLastShown = this.lastShown
+    this.lastShown = Math.max(Math.min(current + this.minItems, this.items.length), this.lastShown)
+    if (oldLastShown < this.lastShown) {
+      this.updateObserved()
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
I don't really know. I use to put the 3 tags for code completion and facilitate modification tasks.
I think we can set a rule for this if it is not set. 

What do you think @y4izus @delapuente @eddybrando

Reply:
The only real difference I'm aware of is that **Vue Loader** will still add the `data-v-<hash>` attributes to the component DOM element and its children, even if the `style` tag is empty, because of the `scoped` attribute. Like this: https://vue-loader.vuejs.org/guide/scoped-css.html#scoped-css

I'd personally prefer to remove the `style` tag form the component file if it's not being used.

Reply:
Agree with @eddybrando